zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -240,6 +241,7 @@ private record CamundaExporterRecordFilter() implements RecordFilter {
             PROCESS_INSTANCE,
             ROLE,
             VARIABLE,
+            VARIABLE_DOCUMENT,
```

Comment:
🔧 I think we need to add `PROCESS_MESSAGE_SUBSCRIPTION` as well, right?

Reply:
Yes, you're right!
I will still merge this one, as people are complaining on variable update operations and will create another PR.

Reply:
Ok, I can't merge as it's not approved. Let me add another type now)

Reply:
Are we missing test coverage, as we haven;t seen this before? Can we add a test for it? 🤔 Or will we cover this when we migrate the existing IT to the exporter?

Reply:
We don't have test coverage for it indeed. The filter is only added to the exporter context and then used in the exporter container and the exporter director. So it's one level up.

We might indeed want to cover it with tests anyway, but it's hard to think how right now.

Reply:
VARIABLE_DOCUMENT and PROCESS_MESSAGE_SUBSCRIPTION will be covered by migrated IT tests!
But I also found GROUP and MAPPING types, that were missing. Not sure whether we have any IT tests for those 🤔